src/components/Table/TablePropTypes.js

```diff
@@ -82,6 +82,7 @@ export const TableColumnsPropTypes = PropTypes.arrayOf(
     name: PropTypes.string.isRequired,
     isSortable: PropTypes.bool,
     width: PropTypes.string, // ex: 150px, or 2rem
+    align: PropTypes.string, // ex: start, center, end
```

Comment:
This should be an enum with 3 options 'start', 'center' and 'end'.

Also we should set the default props on the column to be start so that we don't need to explicitly say 'start' when defining a column.

Reply:
Thanks Sean, we will incorporate your suggestion.

Reply:
Hi Sean, Code is updated based on your review comments, please review.